docs/language/functions.rst

```diff
@@ -279,6 +279,38 @@ constructor.
     for storage rent and there is a minimum deposit needed for the contract. As a result,
     constructors always receive value on Parity Substrate.
 
+Overriding function selector
+____________________________
+
+When a function is called, the function selector and the arguments to the function
```

Comment:
```suggestion
When a function is called, its selector and its arguments
```
Function was repeated three times here.

Reply:
I don't think "its selector" is particularly clear. It's a called a function selector.

Reply:
Also now you repeated its

Reply:
I've rephrased this, please let me know what you think.